test/lib/mayaUsd/render/pxrUsdMayaGL/testProxyShapeLiveSurface.py

```diff
@@ -71,7 +71,7 @@ def setUp(self):
         # Get the viewport widget.
         self._view = OMUI.M3dView()
         OMUI.M3dView.getM3dViewFromModelPanel(self._panel, self._view)
-        self._viewWidget = wrapInstance(long(self._view.widget()), QWidget)
+        self._viewWidget = wrapInstance(int(self._view.widget()), QWidget)
```

Comment:
In py3 long no longer exists. This same change was made when we ported Maya to py3.